SPECIFICATION.md

```diff
@@ -0,0 +1,368 @@
+  "effective": "2023-01-01T00:00:00Z",
+  "versions": ["1.0.0"],
+  "ranges": [],
+  "published": "2023-01-01T00:00:00Z",
+  "modified": "2023-01-01T00:00:00Z",
+  "references": [
+    "https://example.com/release-notes"
+  ]
+}
+```
+
+#### endOfDevelopment
+*Category: Version Event*
+
+Indicates when the manufacturer ceases development of a component or service.
+
+**Additional Required Fields:**
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `versions` | array[string] | Array of simple version strings (e.g., ["1.0.0", "1.1.0"]). Required for version-specific events. |
+| `ranges` | array[string] | Array of VERS-formatted version ranges (e.g., ["vers:npm/>=1.0.0|<2.0.0"]). Required for range-based events. |
+
+Example:
+```json
+{
+  "type": "endOfDevelopment",
+  "effective": "2024-01-01T00:00:00Z",
+  "ranges": ["vers:npm/>=1.0.0|<2.0.0"],
+  "versions": [],
+  "published": "2023-06-01T00:00:00Z",
+  "modified": "2023-06-01T00:00:00Z",
+}
+```
+
+#### endOfSupport
+*Category: Version Event*
+
+Indicates when the manufacturer ceases any and all support of a component or service. This point in time marks a transfer of risk from the manufacturer to the consuming organisation or user of the component or service, encompassing all cybersecurity knowledge and known vulnerabilities, with no further assistance provided by the manufacturer.
+
+**Additional Required Fields:**
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `versions` | array[string] | Array of simple version strings (e.g., ["1.0.0", "1.1.0"]). Required for version-specific events. |
+| `ranges` | array[string] | Array of VERS-formatted version ranges (e.g., ["vers:npm/>=1.0.0|<2.0.0"]). Required for range-based events. |
+
+Example:
+```json
+{
+  "type": "endOfSupport",
+  "effective": "2024-01-01T00:00:00Z",
+  "versions": ["1.0.0", "1.1.0"],
+  "ranges": ["vers:npm/>=2.0.0|<3.0.0"],
+  "published": "2023-06-01T00:00:00Z",
+  "modified": "2023-06-01T00:00:00Z",
+}
+```
+
+#### endOfGuaranteedSupport
+*Category: Version Event*
+
+Indicates when the manufacturer no longer provides assured services such as technical assistance, user training, repairs, spare parts, and software updates for a component or service. Beyond this period, any support offered is discretionary and may incur extra costs or have restrictions.
+
+**Additional Required Fields:**
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `versions` | array[string] | Array of simple version strings (e.g., ["1.0.0", "1.1.0"]). Required for version-specific events. |
+| `ranges` | array[string] | Array of VERS-formatted version ranges (e.g., ["vers:npm/>=1.0.0|<2.0.0"]). Required for range-based events. |
+
+Example:
+```json
+{
+  "type": "endOfGuaranteedSupport",
+  "effective": "2024-01-01T00:00:00Z",
+  "ranges": ["vers:npm/>=1.0.0|<2.0.0"],
+  "versions": [],
+  "published": "2023-06-01T00:00:00Z",
+  "modified": "2023-06-01T00:00:00Z",
+}
+```
+
+#### endOfLife
```

Comment:
endOfSupport and endOfLife feel equivalent to me. what's the difference?

Reply:
updated wording to make it clear the difference